### PR TITLE
Add hidden channel case for channelinfo and pins

### PR DESCRIPTION
**Pull request by GriffinG1**

### CODE REVIEW

File: cogs/serverinfo.py [hunk 1]
```diff
@@ -195,9 +195,12 @@ def find_server(self, msg):
             data.add_field(name="Users", value=userlist)
             data.add_field(name="Bitrate", value=channel.bitrate)
         elif isinstance(channel, discord.TextChannel):
-            pins = await channel.pins()
             data.add_field(name="Members", value="%s"%len(channel.members))
-            data.add_field(name="Pins", value=len(pins), inline=True)
+            try:
+                pins = await channel.pins()
+                data.add_field(name="Pins", value=len(pins), inline=True)
+            except:
```
Comment (LyricLy) [inline cogs/serverinfo.py]: You shouldn't use `except`; it can unintentionally catch errors that you're not expecting.

Comment (GriffinG1) [reply cogs/serverinfo.py]: You shouldn't speak; It can unintentionally give away your idiocy

Comment (GriffinG1) [reply cogs/serverinfo.py]: Fuck you, deleting my comments. @appu1232 lyric's abusing me

**Comment by LyricLy** [comment]
This has been addressed in commit bfb85c2.

File: cogs/serverinfo.py (hunk 1 above)
Comment (chenzw95) [reply cogs/serverinfo.py]: Please explicitly catch HTTPException instead. The presence of other exceptions may due to bugs in code, and we don't want to miss them oht by accident.

**Pull request closed without merging**